cms/djangoapps/contentstore/views/course.py

```diff
@@ -815,6 +819,22 @@ def course_info_update_handler(request, course_key_string, provided_id=None):
     # can be either and sometimes django is rewriting one to the other:
     elif request.method in ('POST', 'PUT'):
         try:
+            if request.method == 'POST':
```

Comment:
Sorry, I just noticed this. Here we should check to see if the feature flag is set.